client-js/queryEditor/QueryEditor.js

```diff
@@ -160,6 +171,11 @@ class QueryEditor extends React.Component {
     const { query } = this.state
     query[field] = value
     this.setState({ query })
+    //console.log(sqlFormatter.format(value))
```

Comment:
Oh forgot to mention once changes have been made could you remove any extra comments used during development?

Reply:
of course  0-0,i wrote about 20 console.log to understand how a request goes in sqlpad:)
I deleted all comments and tracing-logs except this one  Orz!